conf/nginx/nginx.conf

```diff
@@ -1,128 +1,40 @@
+user nginx;
+worker_processes auto;
+
+
 events {
-  worker_connections  4096;
+    worker_connections 1024;
```

Comment:
`worker_connections` is limited to the available simultaneous open files on the VM, which is actually `1024`, so no point in increasing beyond that:
```
[root@designsafe01 ~]# ulimit -a
core file size          (blocks, -c) 0
data seg size           (kbytes, -d) unlimited
scheduling priority             (-e) 0
file size               (blocks, -f) unlimited
pending signals                 (-i) 63374
max locked memory       (kbytes, -l) 64
max memory size         (kbytes, -m) unlimited
open files                      (-n) 1024                               <----- This determines upper limit of worker_connections
pipe size            (512 bytes, -p) 8
POSIX message queues     (bytes, -q) 819200
real-time priority              (-r) 0
stack size              (kbytes, -s) 8192
cpu time               (seconds, -t) unlimited
max user processes              (-u) 63374
virtual memory          (kbytes, -v) unlimited
file locks                      (-x) unlimited
```